### ISSUE TITLE
Continue to fine-tune from wmt20-comet-da checkpoint

### ISSUE THREAD

**Issue by xu1998hz** [Unbabel/COMET#49]
Hi Ricardo, is it possible for me to train from your checkpoint wmt20-comet-da? Currently, I saw that "wmt20-comet-da" contains only the model.

**Comment by ricardorei** [comment]
It is possible but as you said, I don't have the optimizer and trainer state stored. But you can always load the model weights and initialize the rest from scratch.

I don't think this is a major problem since we did not use any fancy scheduler etc..
